doc/upgrade/7.2.md

```diff
@@ -0,0 +1,28 @@
+# Upgrade steps from 7.1 to 7.2
+
+See `doc/bc/changes-7.2.md` for requirements changes and deprecations.
+
+## MySQL/MariaDB database tables character set change
+
+The character set for MySQL/MariaDB database tables is changed from `utf8` to `utf8mb4` to support 4-byte characters. See `data/update/mysql/dbupdate-6.12.0-to-7.2.0.sql` for the SQL upgrade statements.
```

Comment:
data/update/mysql/dbupdate-7.0.0-to-7.2.0.sql

Reply:
Right, thanks. Just making sure if 7.0 is not a typo, since you said 7.1 before...?

Reply:
yes, it should be 7.1 :)